src/main/java/marquez/client/MarquezClient.java

```diff
@@ -45,12 +45,29 @@
 
 @Slf4j
 public class MarquezClient {
+  @VisibleForTesting
+  static final URL DEFAULT_BASE_URL = Utils.toUrl("http://localhost:8080/api/v1");
+
+  @VisibleForTesting static final String DEFAULT_NAMESPACE_NAME = "default";
+
   @VisibleForTesting static final int DEFAULT_LIMIT = 100;
   @VisibleForTesting static final int DEFAULT_OFFSET = 0;
 
   @VisibleForTesting final MarquezHttp http;
   @Getter private final String namespaceName;
 
+  public MarquezClient(final String namespaceName) {
+    this(DEFAULT_BASE_URL, namespaceName);
```

Comment:
could we add a constructor overload that passes in the default namespace name?

Something like this?
```java
  public MarquezClient() {
    this(DEFAULT_BASE_URL, DEFAULT_NAMESPACE_NAME);
  }
```

Reply:
Yes, totally missed that case!